synapse/tests/utils.py

```diff
@@ -669,15 +682,12 @@ def getTestDir(self, mirror=None):
                     srcpath = self.getTestFilePath(mirror)
                 dstpath = os.path.join(tempdir, 'mirror')
                 shutil.copytree(srcpath, dstpath)
-                s_scope.set('dirn', dstpath)
```

Comment:
pulling dirn out of the scope means that someone calling into something like `getTestDmon()` doens't have a easy way to pull the test dirn used by that guy.

Reply:
dmon.dirn ( which also shouldn't be used by subsequent test code that crates junk files anyway really )